File: apps/problems/Binary Search/boilerPlate-Full/function.js

  ##USER_CODE_HERE##
  
  const input = require('fs').readFileSync('/dev/stdin', 'utf8').trim().split('\n').join(' ').split(' ');
  const k = parseInt(input.shift());
  const result = binarysearch(k);
  console.log(result);
      

File: apps/problems/Majority Element/boilerPlate-Full/function.rs

  use std::io::{self, BufRead};
  
  ##USER_CODE_HERE##
  
  fn main() {
    let stdin = io::stdin();
    let mut input = stdin.lock().lines().map(|l| l.unwrap());
    let k: i32 = input.next().unwrap().parse().unwrap();
    let result = binarysearch(k);
    println!("{}", result);
  }
      